components/layout/navbar/navbar.tsx

```diff
@@ -88,7 +88,7 @@ const Navbar = ({ DesHeight }) => {
         </div>
         <Link shallow href="/">
           <a className={className}>
-            <Img className="inner-logo" src="/images/logo-2.0.svg" />
+            <Img layout="fill" className="inner-logo" src="/images/logo-2.0.svg" />
```

Comment:
What layout="fill" is for?

Reply:
without it i get this error
![image](https://user-images.githubusercontent.com/89342835/176996613-878618be-9eb0-4003-b195-b9e1910c9de1.png)
Also happens in master